T2CC_APP/t2cc/app/src/main/res/layout/activity_home.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This was just a stub to get LogOutActivity to build. It will be replace by the contents of PR #7